docs/examples/clipping.py

```diff
@@ -0,0 +1,74 @@
+"""
+Clipping Detection
+===================
+
+Identifying clipping periods using the PVAnalytics clipping module.
+"""
+
+# %%
+# Identifying and removing clipping periods from AC power time series
+# data aids in generating more accurate degradation analysis results,
+# as using clipped data can lead to over-predicting degradation. In this
+# example, we show how to use
+# :py:func:`pvanalytics.features.clipping.geometric`
+# to mask clipping periods in an AC power time series. We use a
+# normalized time series example provided by the PV Fleets Initiative,
+# where clipping periods are labeled as True, and non-clipping periods are
+# labeled as False. This example is provided taken from the DuraMAT DataHub:
```

Comment:
There might be an extra word in here

Reply:
Fixed in 
commit 4f379e6e77b965fd72abbe33c477ab7c41d7a153